### DISCUSSION TITLE
Welcome to ADBugger Discussions!

### DISCUSSION THREAD

**Discussion by Pulimet** [Pulimet/ADBugger#30] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other community members. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, please feel free to comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by venu1394** [comment]
How to install and launch ADBugger? Can you please provide steps.

**Reply by Pulimet** [reply]
Sure. Which OS? Mac/Windows?

**Reply by venu1394** [reply]
Wiindows

**Reply by Pulimet** [reply]
1. Open https://github.com/Pulimet/ADBugger/releases
2. Go to latest release
3. Open Assets
4. Download ADBugger-1.0.5.msi 
5. Launch it

**Reply by venu1394** [reply]
Installed and launched the App. Thanks. I will explore more.

**Reply by Pulimet** [reply]
You are welcome.
